src/test/unit_tests/test_nb_native_fs.js

```diff
@@ -47,6 +65,11 @@ mocha.describe('nb_native fs', function() {
                 _.omit(res, 'xattr', 'mtimeNsBigint', 'atimeNsBigint', 'ctimeNsBigint'), // we need to remove xattr, mtimeSec, mtimeNsec from fs_napi response as the node JS stat doesn't return them
                 _.omitBy(res2, v => typeof v === 'function'),
             );
+
+            assert.notDeepStrictEqual(
```

Comment:
I would add a comment about this check and the check above so it would be clearer.